src/components/Shared/DatePicker.vue

```diff
@@ -36,16 +36,25 @@
 		:use12h="showAmPm"
 		:append-to-body="appendToBody"
 		v-bind="$attrs"
+		confirm
 		v-on="$listeners"
 		@close="close"
 		@change="change"
-		@pick="pickDate"
-		confirm>
+		@pick="pickDate">
 		<template #icon-calendar>
-			<button class="datetime-picker-inline-icon icon"
-				:class="{'icon-timezone': !isAllDay, 'icon-new-calendar': isAllDay, 'datetime-picker-inline-icon--highlighted': highlightTimezone}"
+			<Button type="tertiary-no-background"
 				@click.stop.prevent="toggleTimezonePopover"
-				@mousedown.stop.prevent="() => {}" />
+				@mousedown.stop.prevent="() => {}">
+				<template #icon>
+					<IconNewCalendar v-if="isAllDay"
+						:size="20" />
+					<IconTimezone v-else-if="highlightTimezone"
+						class="highlighted-timezone-icon"
+						:size="20" />
+					<IconTimezone v-else-if="!highlightTimezone"
+						:size="20" />
```

Comment:
```suggestion
					<IconTimezone v-else
						:class="{ 'highlighted-timezone-icon': highlightTimezone }"
						:size="20" />
```
Sorry, thought of it afterwards.

Reply:
ok, i will change it, but it does work like it is though

Reply:
Yes, the class will be applied only if `highlightTimezone` is `true`.

Reply:
done